packages/contracts/.compiler.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "0.8.10",
+  "settings": {
+    "optimizer": {
+      "enabled": true,
+      "runs": 20000
+    }
+  }
+}
```

Comment:
Delete if unused

Reply:
It's used in [hardhat.config.ts](https://github.com/TrueFiEng/devcon-raffle/blob/setup-project/packages/contracts/hardhat.config.ts)